tests/Containerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/centos/centos:latest
+
+RUN true \
+ && dnf -y install python3-pip \
+ && pip3 install jenkins-job-builder \
+ && dnf -y clean all \
+ && true
+
+COPY . /opt/build/
+
+# make jobs directory writable, otherwise deploy-centos-ci.sh fails
+RUN chmod ugo=rwx /opt/build/jobs
+
+ENV HOME=/opt/build
```

Comment:
For what purpose are we setting `HOME` environment variable?

Reply:
not sure, I took it from the Ceph-CSI container image... maybe setting `WORKDIR` is sufficient?

Reply:
`WORKDIR` should be enough.

Reply:
`WORKDIR` should be enough I guess.

Reply:
Removed `HOME` now.